addons/mail/models/mail_message.py

```diff
@@ -1032,6 +1040,17 @@ def _notify_message_notification_update(self):
     # TOOLS
     # ------------------------------------------------------
 
+    def _get_message_preview(self, max_char=190):
+        """Returns an unformatted version of the message body. Unless `max_char=0` is passed,
+        output will be capped at max_char characters with a ' [...]' suffix if applicable.
+        Default `max_char` is the longest known mail client preview length (Outlook 2013)."""
+        self.ensure_one()
+
+        plaintext_ct = html2text_only(self.body or '').strip()
```

Comment:
IMO `html2text_only` should be able to receive False as a value, so you don't need this here